docs/en/02_configuration.md

```diff
@@ -96,7 +97,7 @@ MyProject\MyApp\Product:
     - SilverStripe\Forager\Extensions\SearchServiceExtension
 ```
 
-DataObjects also require the `SilverStripe\Versioned\Versioned` extension. Non-versioned content is not yet supported. By default a versioned object will be added to the index when it is published and removed when it is unpublished.
+Most DataObjects you use will also have `SilverStripe\Versioned\Versioned` extension (e.g. SiteTree). By default a versioned object will be added to the index when it is published and removed when it is unpublished. The [SearchServiceExtension](../../src/Extensions/SearchServiceExtension.php) class is responsible for listing to these events.
 
```

Comment:
Somewhere on this page (further down I think) it mentions "It is important to note that the keys of fields can be named anything you like". I'm not sure, do we need to mention any reserved ones that the internals will create. I'm thinking record_id, record_base_class, source_class. 🤷

Reply:
good point I'll add them